pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -71,8 +71,13 @@ func (configgen *ConfigGeneratorImpl) BuildClusters(env *model.Environment, prox
 			return nil, err
 		}
 
+		serviceAccounts, err := getServiceAccounts(env, proxy)
+		if err != nil {
+			return nil, err
```

Comment:
Print out the error for debugging purpose?

Reply:
log error in getServiceAccounts